src/HotChocolate/Core/src/Types.Scalars/LatitudeType.cs

```diff
@@ -0,0 +1,167 @@
+using System;
+using System.Globalization;
+using System.Text.RegularExpressions;
+using HotChocolate.Language;
+
+namespace HotChocolate.Types
+{
+    /// <summary>
+    /// The `LatitudeType` scalar represents a valid decimal degrees latitude number.
+    /// <a href="https://en.wikipedia.org/wiki/Latitude">Read More</a>
+    /// </summary>
+    public class LatitudeType : ScalarType<double, StringValueNode>
+    {
+        /// <summary>
+        /// Initializes a new instance of <see cref="LatitudeType"/>
+        /// </summary>
+        public LatitudeType()
+            : this(
+                WellKnownScalarTypes.Latitude,
+                ScalarResources.LatitudeType_Description)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="LatitudeType"/>
+        /// </summary>
+        public LatitudeType(
+            NameString name,
+            string? description = null,
+            BindingBehavior bind = BindingBehavior.Explicit)
+            : base(name, bind)
+        {
+            Description = description;
+        }
+
+        /// <inheritdoc />
+        public override IValueNode ParseResult(object? resultValue)
+        {
+            return resultValue switch
+            {
+                null => NullValueNode.Default,
+
+                string s when Latitude.TryDeserialize(s, out var value) => ParseValue(value),
+
+                double d => ParseValue(d),
+
+                int d => ParseValue(d),
+
+                _ => throw ThrowHelper.LatitudeType_ParseValue_IsInvalid(this)
+            };
+        }
+
+        /// <inheritdoc />
+        protected override double ParseLiteral(StringValueNode valueSyntax)
+        {
+            if (Latitude.TryDeserialize(valueSyntax.Value, out var value) && value is not null)
+            {
+                return value.Value;
+            }
+
+            throw ThrowHelper.LatitudeType_ParseLiteral_IsInvalid(this);
+        }
+
+        /// <inheritdoc />
+        protected override StringValueNode ParseValue(double runtimeValue)
+        {
+            if (runtimeValue is > Latitude._min and < Latitude._max)
```

Comment:
This check should be in the TrySerialize method. It should return false if the check didn't succeed